indico/htdocs/sass/base/_layout.scss

```diff
@@ -136,7 +136,14 @@ body > * {
     }
 }
 
-/* Flexbox layout */
+
+// -- Flexbox layout -----------------------------------------------------------
```

Comment:
And yes, maybe a different file wouldn't be such a bad idea. OTOH it's the `layout.scss`, so the flexbox classes fit just fine there.